docs/source/distributions.rst

```diff
@@ -522,6 +522,13 @@ SoftplusLowerCholeskyTransform
     :undoc-members:
     :show-inheritance:
 
+UnitLowerCholeskyTransform
```

Comment:
It looks like we're trying to keep this file alphabetical, can you try to preserve that convention?

Reply:
done